### ISSUE TITLE
All the latest nightlies crash on startup on Tolino Epos 3

### ISSUE THREAD

**Issue by I2ujbNGakP8KfLKbho0Z** [koreader/koreader#12613]
* KOReader version: v2024.07-237-g92bcbe06f_2024-10-07
* Device: Tolino Epos 3

#### Issue

KOReader crashes on start with all the latest nightlies. Performing a clean install doesn't help. Trying to start the app multiple times doesn't help. The last one i've tested and that has worked is [this ](https://github.com/koreader/koreader/issues/12348#issuecomment-2322865601) version mentioned in #12348 . Help would be appreciated!

#### Steps to reproduce

- Perform a clean install of a recent KOReader nightly build (newest i've tested was [koreader-android-arm-v2024.07-237-g92bcbe06f_2024-10-07.apk](http://build.koreader.rocks/download/nightly/v2024.07-237-g92bcbe06f_2024-10-07/koreader-android-arm-v2024.07-237-g92bcbe06f_2024-10-07.apk))
- Open KOReader

##### `crash.log` (if applicable)
crash.log wasn't generated, but here's the output of `adb logcat 'KOReader:I MainActivity:V NativeGlue:V NativeThread:V ActivityManager:W AndroidRuntime:E DEBUG:* *:F'` (command taken from [here.](https://github.com/koreader/koreader/issues/10395#issuecomment-1537131597))

[crash_logcat.log](https://github.com/user-attachments/files/17297822/crash_logcat.log)

**Comment by benoit-pierre** [comment]
Please use our [logcat Python script](https://github.com/koreader/koreader/blob/master/tools/logcat.py) to get a clean crash log:
- uninstall koreader
- reinstall koreader
- clear the logcat buffer: `adb logcat -c`
- start koreader
- run `./tools/logcat.py -d >crash.log` (alternatively, if you don't have Python / can't use the script, use `adb logcat -d >crash.log`)

**Comment by pazos** [comment]
Can you reproduce, @dboleslawski ?

**Comment by I2ujbNGakP8KfLKbho0Z** [comment]
Here's a cleaned crash log created with the script @benoit-pierre :
[crash.log](https://github.com/user-attachments/files/17298750/crash.log)

**Comment by benoit-pierre** [comment]
When loading plugins, I suspect this might be the terminal plugin acting up again…

**Comment by I2ujbNGakP8KfLKbho0Z** [comment]
2024.07-232 didn't work either, @pazos
Here's a crash log from that version:
[crash.log](https://github.com/user-attachments/files/17298840/crash.log)

**Comment by benoit-pierre** [comment]
Can you try editing `/sdcard/koreader/settings.reader.lua` to disable it?

```lua
    ["plugins_disabled"] = {
        ["terminal"] = true,
    },
```

**Comment by I2ujbNGakP8KfLKbho0Z** [comment]
The version linked [here](https://github.com/koreader/koreader/issues/12601#issuecomment-2395430915) starts and works.

**Comment by benoit-pierre** [comment]
And that version is missing the terminal plugin (nuked from orbit).

**Comment by I2ujbNGakP8KfLKbho0Z** [comment]
> Can you try editing `/sdcard/koreader/settings.reader.lua` to disable it?
> 
> ```lua
>     ["plugins_disabled"] = {
>         ["terminal"] = true,
>     },
> ```

Tried adding this to the settings file created with a working version and pasting that into the koreader directory created by the latest nightly and that makes it work, thanks!

**Comment by NiLuJe** [comment]
Huh.

@benoit-pierre: Your os.exec patch in android-luajit-launcher sounded sensible to me, feel free to PR that, and I'll try to cobble the plugin PR into submission this WE, because this is becoming somewhat silly ;p.

**Comment by benoit-pierre** [comment]
@I2ujbNGakP8KfLKbho0Z: can you check [this version](https://github.com/user-attachments/files/17318544/koreader-android-arm-dev.zip) fix the terminal plugin issue?

**Comment by benoit-pierre** [comment]
Closed by #12630.
